src/actions.js

```diff
@@ -43,6 +43,36 @@ export function validateClaimCode(code) {
   return graphql(payload, "CLAIM_CLAIM_CODE_COUNT");
```

Comment:
Where do we use validateClaimCode? I saw that you deleted the function which used this action. This was the old approach for claim code validation (correct me if I'm wrong). If so, we should delete all the residues after it (including this unneccesary action).